src/main/java/io/github/fixitlater/quizapi/service/QuestionService.java

```diff
@@ -55,20 +41,13 @@ private QuestionEntity getRandomQuestionFromList(List<QuestionEntity> questionEn
         return questionEntityMap.get(randomKey);
     }
 
-    public QuestionWithAnswersDTO getRandomQuestionFromCategory(String category) {
-        return QuestionWithAnswersDTO.fromEntity(getRandomQuestionFromList(questionRepository.findByCategory(category)));
-    }
-
-    public QuestionWithAnswersDTO getRandomQuestionFromLanguage(Language language) {
-        return QuestionWithAnswersDTO.fromEntity(getRandomQuestionFromList(questionRepository.findByLanguage(language.name())));
-    }
-
     public QuestionWithAnswersDTO getRandomQuestionByCategoryAndOrLanguage(Category category, Language language) throws NoSuchElementException {
         String languageString = ((language == Language.ANY) ? "%" : language.name());
         String categoryString = ((category == Category.ANY) ? "%" : category.name());
         try {
             return QuestionWithAnswersDTO.fromEntity(getRandomQuestionFromList(questionRepository.findByCategoryAndOrLanguage(categoryString, languageString)));
         } catch (NoSuchElementException e){
+            e.printStackTrace();
```

Comment:
Jak używacie Lomboka to użyjcie adnotacji `@SL4J`
Później w kodzie możecie logować rzeczy przy użyciu `log.error("Something went wrong {}", e);`